### ISSUE TITLE
Can't compile Redis crate since combine 4.5.0  released 

### ISSUE THREAD

**Issue by eyalsatori** [Marwes/combine#311]
Hi, 

I have code that use the Redis crate that import combine. 

Since the 4.5.0 released the code could not compile : 

```
error[E0433]: failed to resolve: use of undeclared type or module bytes_05

   --> /Users/eyal/.cargo/registry/src/github.com-1ecc6299db9ec823/combine-4.5.0/src/stream/buf_reader.rs:419:6
    |  419 
    | impl bytes_05::BufMut for Bytes05<'_> {
    |      ^^^^^^^^ use of undeclared type or module bytes_05

error[E0433]: failed to resolve: use of undeclared type or module bytes_05

   --> /Users/eyal/.cargo/registry/src/github.com-1ecc6299db9ec823/combine-4.5.0/src/stream/buf_reader.rs:606:26
    | 606
    |     fn poll_write_buf<B: bytes_05::Buf>(
    |                          ^^^^^^^^ use of undeclared type or module bytes_05

```

Don't sure if the fix need be here , or in Redis crate,  or in my code . 
But i will be happy to get some help in order to resolve this.  

thanks a lot..

**Comment by eyalsatori** [comment]
well , my problem was that ib redis crate I used  `tokio-rt-core` that supported until 0.17 - so my app stuck with that version,
when i switched to `tokio-comp` i was able to upgrade redis 0.19 , and then it's compiled with combine 4.5

**Comment by Marwes** [comment]
Fixed in combine 4.5.1
